src/avt/QtVisWindow/vtk/vtkRubberBandMapper2D.C

```diff
@@ -394,5 +401,173 @@ vtkRubberBandMapper2D::RenderOverlay_Qt(vtkViewport* viewport, vtkActor2D* actor
     double* actorColor = actor->GetProperty()->GetColor();
     SET_FOREGROUND_D(actorColor);
 
-#include <vtkRubberBandMapper2D_body.C>
+    int numPts;
+    vtkPolyData *input= vtkPolyData::SafeDownCast(this->GetInput());
```

Comment:
While moving this from vtkRubberBandMapper2D_body.C might make things readable, the reason this code section is in a separate file is to reduce code duplication. It is used by this file on line 320 as well as the location you removed it from. It is also used by vtkDashedXorGridMappe2D.  Please restore vtkRubberBandMapper2D_body.C and remove this section.

Reply:
That's odd... I did a grep, and I thought it only showed up in this file. I'll add it back. It would be nice if this was wrapped in some kind of function or other standard method of distributing code, though.

Reply:
Fixed.